package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "faq-chatbot-edit",
+  "version": "1.0.0",
+  "description": "A chatbot using API.ai.  This is a reusable package to integrate into tools and sites where needed.",
+  "main": "scripts.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": ""
```

Comment:
This needs a run argument

Reply:
Have a look at the Eslint homepage, they'll typically have a command like: `eslint js/*.js` and optional `--config .eslintrc` if not in your root directory

Reply:
added 
    `"eslint": "eslint ./*.js"`
into the scripts, and it seems to be linting, but is giving me an error message at the end:
`npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! faq-chatbot-edit@0.0.0 eslint: `eslint ./*.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the faq-chatbot-edit@0.0.0 eslint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/janietito/.npm/_logs/2017-09-10T10_25_16_279Z-debug.log`